plugins/mentions.md

```diff
@@ -54,14 +54,12 @@ tinymce.init({
 });
 ```
 
-> **Important:**  The values returned in the users array for "id" and "name" all need to be `String` values.
-
 The `success` callback can be passed an optional array of extra items. When clicked, the menu reloads and passes additional query parameters to the fetch function. The extra items can be used to search with different queries or show additional results, such as a full text search (which is slower to fetch). Each extra item should contain:
 
 * A "text" property for the content to be displayed in the menu.
 * A "meta" property for that will be passed using the fetch query parameter.
 
-#### Example with extras
+##### Example with extras
```

Comment:
Why is this a h5? I may not have enough context here but it looks like it should have stayed as a h4.

Reply:
The "extras" thing is part of `mentions_fetch` which is documented under a h4 above, so I see this as a subsection of that.

Reply:
Fixed this